orca/topology/alerts/extractor.py

```diff
@@ -88,9 +89,13 @@ def _mapping(self):
         return self.__mapping
 
     def map(self, name, labels):
-        mapping = self._mapping.get(name)
+        mapping = self._mapping['plain'].get(name)
         if not mapping:
-            raise exceptions.MappingNotFound(key=name)
+            try: 
+                key = [regex for regex in self._mapping['regex'].keys() if re.compile(regex).match(name)][0]
```

Comment:
As far as I remember, it is not necessary to compile a regex for each match. Internally, Python compiles the regex and maintains it in a cache.

The [match](https://docs.python.org/3/library/re.html#re.match) method should do the thing:

```python
re.match(regex, name)
```